nav2_smac_planner/include/nav2_smac_planner/analytic_expansion.hpp

```diff
@@ -119,7 +119,7 @@ class AnalyticExpansion
    * @return The score of the refined path
    */
   float refineAnalyticPath(
-    const NodePtr & current_node,
+    NodePtr & node,
```

Comment:
Add doxygen that this node head may change as a result of refinement

Reply:
done in https://github.com/ros-navigation/navigation2/pull/5198/commits/de6b5b24897f06ba27a78817352db3f2d64d7f0a